app/src/main/java/com/example/bookwormadventuresdeluxe2/EditBooksActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.bookwormadventuresdeluxe2.Utilities.Status;
+import com.google.zxing.integration.android.IntentIntegrator;
+import com.google.zxing.integration.android.IntentResult;
+
+public class EditBooksActivity extends AppCompatActivity
```

Comment:
I thought this was for adding a book, not editing one. Did you call it this so the activity can be reused for editing? If thats the case I would call it AddOrEditBooksActivity so the intention is clear

Reply:
Ok ty